### PR TITLE
Support unicode

### PR DESCRIPTION
**Pull request by plowman**
My script broke because I had non-ascii characters. This fixes it.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,21 +9,6 @@
 # Search API URL for reference: https://sprint.ly/api/items/search.json?q=tag:spirit-112
 # Requires basic auth, with Sprintly API token as "password", email as "username"
 
-class File(object):
-  """
-  Use a file object to catch errors
-  """
-
-  def __init__(self, file_name, method):
-    self.file_obj = open(file_name, method)
```
Comment (gkadillak) [inline main.py]: why not just use `codes.open(file_name, method, encoding='utf-8')` here?

Comment (plowman) [inline main.py]: Why have a File class in the first place?

Comment (gkadillak) [inline main.py]: Have more control about file opening, closing, throwing errors because of the context manager being used.

Comment (bsans) [inline main.py]: I think the portability of the code is higher if there is no `File` class to work as a context manager.  And less potential for bugs.

**Comment by gkadillak** [comment]
The deed has been done

**Comment by bsans** [comment]
🎉

**Pull request merged**